lib/coins/zec.js

```diff
@@ -32,5 +38,6 @@ function depositUrl (address, amount) {
 function validate (network, address) {
   if (!network) throw new Error('No network supplied.')
   if (!address) throw new Error('No address supplied.')
-  return base58Validator(network, address, base58Opts)
+  if (base58Validator(network, address, base58Opts)) return true
+  if (zecBech32Validator(network, address, bech32Opts)) return true
```

Comment:
Need to return ``false`` if the other tests fail. Also do we have a test for failing this validation function? It should have caught that.